docs/dev-concepts/participant-authentication.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_label: Participant authentication
+sidebar_position: 5
+---
+
+# Participant authentication with XMTP
+
+<!--What do we think about the title of this topic? Consider this definition of "participant" - "Topics have participants, but not users. Client apps and the XMTP network have users, but not participants." Based on this definition I don't think we are authenticating participants, are we? I am thinking of "User authentication" (friendly language) or "Blockchain account authentication" (less friendly, but more accurate?)-->
+
+<!--I removed the language about relationships between XMTP identities being established using a set of keys because the XMTP identity IS one of these keys. The way this is worded, if the relationship is established by the keys, the relationship must be between something that exists before key generation. Following the comment above, I think the keys enable establishing relationships between users (sender and recipient), or more specifically, the sender blockchain account and recipient blockchain account. Any thoughts?-->
```

Comment:
Yeah, I think saying that the IdentityPublicKey is the XMTP identity is probably the simplest way to go about it.

Reply:
Although if the identity key gets replaced with a new one, we're not creating a new identity. The identity is still tied to the blockchain account address.

Reply:
Ah - such an important point, @mkobetic - got it. I think need to stop equating the IdentityPublicKey with the XMTP identity. May I ask you... is the XMTP identity generated along with the IdentityPublicKey and IdentityPrivateKey during the Create Identity one-time step? Conceptually, would you say that the XMTP identity is the immutable value that provides the permanent tie between mutable identity keys and the blockchain account address? We may not need to put a lot of attention on it, but I think we need to define it only because the signature flow clearly references "Create Identity" and "Enable Identity" - so we need to describe what the Identity is. Does the client or the client protocol do anything with the XMTP identity value?